x/mongo/driver/topology/rtt_monitor.go

```diff
@@ -285,7 +297,7 @@ func (r *rttMonitor) Min() time.Duration {
 	r.mu.RLock()
 	defer r.mu.RUnlock()
 
-	return r.minRTT
+	return r.min()
```

Comment:
Calculating the minimum RTT when `Min` is called can create a performance issue. We expect `Min` to be called once for every operation, which could be thousands of times per second. In contrast, we expect new samples to be added about once every 10 seconds. Because of that, it's better to calculate the updated minimum RTT when a new sample is added instead of when `Min` is called.